docs/get_started.md

```diff
@@ -110,32 +110,51 @@ the creation of a new management pack project.
     Along with additional language-specific configuration.
     Currently, `mp-init` offers the following languages:
 
-    `Python`
+    - `Python`
  
-    `Java` 
+    - `Java` 
+ 
+9.  === "Python"
+ 
+        a. `Select a template for your project`
```

Comment:
Can we keep the numbering here consistent? It looks strange the way it's currently laid out:
<img width="774" alt="image" src="https://github.com/vmware/vmware-aria-operations-integration-sdk/assets/3310870/499374b3-b78c-44c1-9750-afc6fe64bfe8">

I would expect the selector itself to be un-indented, and then inside an ordered list with numbers, starting at 9.

Reply:
It seems like this might not be possible out of the box, but I found a plugin that purports to allow it: https://github.com/paulmelis/pm_attr_list

Which I found from this issue:
https://github.com/mkdocs/mkdocs/discussions/2838